### ISSUE TITLE
Closing braces being flushed all the way to the left for inline component props

### ISSUE THREAD

**Issue by riwu** [eslint/eslint#11021]
**Tell us about your environment**

* **ESLint Version:**
```
"eslint": "^5.7.0",
"eslint-config-airbnb": "^17.1.0",
"eslint-plugin-import": "^2.14.0",
"eslint-plugin-jsx-a11y": "^6.1.2",
"eslint-plugin-react": "^7.11.1",
```

* **Node Version:** v8.12.0
* **yarn Version:** 1.9.4

**What parser (default, Babel-ESLint, etc.) are you using?** babel-eslint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "root": true,
  "extends": "airbnb",
  "parser": "babel-eslint",
  "rules": {
    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
    "react/prop-types": 0,
    "no-console": 0,
    "no-debugger": 0,
    "react/sort-comp": 0
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

![](https://i.imgur.com/sIupyet.gif)
```js
const Component = () => (
  <View
    ListFooterComponent={
      <View
        rowSpan={3}
        placeholder="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do"
      />
    }
  />
);
```

```bash
eslint --fix
```

**What did you expect to happen?**
It should be formatted like this instead:
```
const Component = () => (
  <View
    ListFooterComponent={(
      <View
        rowSpan={3}
        placeholder="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do"
      />
    )}
  />
);
```
On the older versions, it will not flush the brackets to the left:
```
"eslint": "^4.13.0",
"eslint-config-airbnb": "^16.1.0",
"eslint-plugin-import": "^2.8.0",
"eslint-plugin-jsx-a11y": "^6.0.2",
"eslint-plugin-react": "^7.5.1",
```

**What actually happened? Please include the actual, raw output from ESLint.**
Line 11's closing braces are being flushed to the left when `eslint` formats it.

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report. You mentioned that "Line 11's closing braces are being flushed to the left when prettier-atom formats it" -- are you sure this problem is related to ESLint? (Does the same thing happen when you run `eslint --fix` on the command line?)

**Comment by riwu** [comment]
@not-an-aardvark I noticed that `eslint --fix` does not flush it to the left again if I manually correct the indentation (that behavior seems to be from `prettier-atom`). However, it still flushes it to the left for the initial `--fix`. Please see the updated screenshot above.

If I run without `--fix`, it gives an error:
> 7:7  error  Missing parentheses around multilines JSX  react/jsx-wrap-multilines
>✖ 1 problem (1 error, 0 warnings)
> 1 error and 0 warnings potentially fixable with the `--fix` option.

**Comment by not-an-aardvark** [comment]
Since the error before using `--fix` is from the `react/jsx-wrap-multilines` rule, I think this is a bug in [`eslint-plugin-react`](https://github.com/yannickcr/eslint-plugin-react). I would recommend reporting an issue in that repository instead.
